accelerator/config.c

```diff
@@ -78,6 +79,7 @@ status_t ta_config_default_init(ta_config_t* const info, iota_config_t* const ta
                                 iota_client_service_t* const service) {
   status_t ret = SC_OK;
   char mss_tmp[] = "/tmp/XXXXXX";
+  UNUSED(mss_tmp);
```

Comment:
Validate the necessity of `mss_tmp`. @HowJMay, can you comment this?

Reply:
we can remove variable `mss_tmp` now